docroot/themes/humsci/humsci_basic/src/scss/settings/_variables.colorful-pairings.scss

```diff
@@ -0,0 +1,61 @@
+// Set the default color pairing value
+// The default color pairing also serves as the fallback color
+// pairing for browsers that do not support CSS custom properties
+$hb-colorful-default: "ocean";
+$hb-colorful-variation: "colorful-teal"; // WIP!!! We may or may not need both $hb-colorful-default AND $hb-colorful-variation
```

Comment:
`$hb-colorful-variation` cannot be removed until the CSS refactor in [JIRA STN-202](https://sparkbox.atlassian.net/browse/STN-202) is complete.